installer/charts/rhtap-acs-scanner/scripts/test-acs-image-scan.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+set -x
+
+ROX_CENTRAL_ENDPOINT="$(kubectl get secrets -n rhtap rhtap-acs-integration -o jsonpath='{.data.endpoint}' | base64 -d)"
+ROX_API_TOKEN="$(kubectl get secrets -n rhtap rhtap-acs-integration -o jsonpath='{.data.token}' | base64 -d)"
+export ROX_CENTRAL_ENDPOINT
+export ROX_API_TOKEN
+
+IMAGE="${IMAGE:-quay.io/fedora/fedora:36-x86_64@sha256:d6e4c7d6d1eaa24d71c8efd7432890acdc0179502224d0aaad6bb05d15ffde19}"
```

Comment:
Oh?

Reply:
I understand it's only used for testing the installation, but can we use an image from registry.redhat.io instead?

Reply:
Unfortunately that's more complicated than just switching the image, as ACS would need to be setup to be integrated with registry.redhat.io (it's basically currently returning an error saying it can't get the image from that registry).

I've chosen as close to possible to a "Red Hat" image by using a Fedora image hosted on quay, which are 2 products strongly linked to our brand. Let me know if you don't think that's good enough.

Reply:
Thanks for looking into it! I think it's fine because:

1. The contents of the fedora image are never executed.
2. The image comes from a registry that is managed by Red Hat.